src/lib/components/Circos.react.js

```diff
@@ -391,30 +402,30 @@ Circos.propTypes = {
             type: PropTypes.oneOf(TRACK_TYPES),
 
             /**
-	 * Specify what data for tooltipContent is
-	 * displayed.
-	 *
-	 * The entry for the "name" key, is any of the keys used in the data loaded into tracks.
-	 * Ex: "tooltipContent": {"name": "block_id"},
-	 *
-	 * To display all data in the dataset use "all" as the entry for the key "name".
-	 * Ex: "tooltipContent": {"name": "all"}
-	 *
-	 * Ex: This will return (source) + ' > ' + (target) + ': ' + (targetEnd)'.
-	 * "tooltipContent": {
-	    "source": "block_id",
-	    "target": "position",
-	    "targetEnd": "value"
-				},
-	 * Ex: This will return (source)(sourceID) + ' > ' + (target)(targetID) + ': ' (target)(targetEnd)'.
-	 * "tooltipContent": {
-	    "source": "source",
-	    "sourceID": "id",
-	    "target": "target",
-	    "targetID": "id",
-	    "targetEnd": "end"
-	}
-	 **/
+             * Specify what data for tooltipContent is
```

Comment:
Do these whitespace changes come from your linter?

Reply:
No i made these changes. The whitespace looked to be messed up so i corrected it.